src/WebJobs.Script/Workers/Rpc/RpcWorkerChannel.cs

```diff
@@ -55,8 +55,8 @@ internal class RpcWorkerChannel : IRpcWorkerChannel, IDisposable
         private ILogger _workerChannelLogger;
         private IMetricsLogger _metricsLogger;
         private IWorkerProcess _rpcWorkerProcess;
-        private TaskCompletionSource<bool> _reloadTask = new TaskCompletionSource<bool>();
-        private TaskCompletionSource<bool> _workerInitTask = new TaskCompletionSource<bool>();
+        private TaskCompletionSource<bool> _reloadTask = new TaskCompletionSource<bool>(TaskCreationOptions.RunContinuationsAsynchronously);
```

Comment:
So did we determine that this does help? I thought it was just helping that one measurement, which wasn't as important as the actual Specialization...

Reply:
There is no overall impact of this change on cold start ( not significant) on my private stamp. This PR fixes the latency reported.